### PR TITLE
Add deploy PR review workflow

### PR DESCRIPTION
**Pull request by gabrielmocanu**
Add deploy PR review workflow

### CODE REVIEW

**Comment by gabrielmocanu** [comment]
The link is: https://security-summer-school.github.io/pr-preview/pr-17/

**Review by teodutu — CHANGES_REQUESTED** [review]
The `.yml` file looks good AFAICT, but wy modify the `contributor-guideline.md` file?

File: content/en/docs/contribution-guidelines/contributor-guideline.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: >
   Make sure you follow them whenever you make a contribution.
 ---
 
-## First Steps
+## First Step
```
[the remark this answers is not in the document]
Comment (gabrielmocanu) [reply content/en/docs/contribution-guidelines/contributor-guideline.md:11]: It was just a test to see how the PR preview looks.
I will revert this change.

[the remark this answers is not in the document]
Comment (teodutu) [reply content/en/docs/contribution-guidelines/contributor-guideline.md:11]: I see.

**Review by teodutu — APPROVED** [review]

**Comment by gabrielmocanu** [comment]
I will close this PR since we won't need any PR to this repository.
All the updated are made through the bot when changes are in the main repositories.

**Pull request closed without merging**